### DISCUSSION TITLE
Can't install - not doing something correctly. HELP

### DISCUSSION THREAD

**Discussion by mikeatencio** [PodcastGenerator/PodcastGenerator#671] [Q&A]
**System Information**
Podcast Generator Version: 
Webserver: 
PHP Version:

**Podcast Generator Config**

```
INSERT THE **CENSORED** CONFIG HERE
``` 

**Description of your problem:**
I can't figure out how to get it to install. Files are there but it won't install. This is the link I'm using, which I'm assuming is wrong: 
http://projectafterlife.net/podcastgen

I uploaded everything. I tried **http://projectafterlife.net/podcastgenerator/** Where do I point it to inside the folder?
Thanks
Mike

**Comment by charabaruk** [comment]
The files in the PodcastGenerator/ directory within the zip file should be placed in the directory of your website where you want PG to be installed. For example, if your website is at ~/mywebsite.com and you want PG to be available at mywebsite.com/podcast, you'd need to create the ~/mywebsite.com/podcast directory and then copy the contents of PodcastGenerator/ into that directory.
